### ISSUE TITLE
Can you share hyperparameters for training your method on PACS dataset?(Source2Target)

### ISSUE THREAD

**Issue by hyeonwoocho7** [MattiaLitrico/Guiding-Pseudo-labels-with-Uncertainty-Estimation-for-Test-Time-Adaptation#7]
Can you provide the hyperparameters (epoch, learning rate, batch size) for applying your method to the PACS dataset?
You have the PACS training code in your repository, but the hyperparameters are not specified only about PACS. I've reached out to you for that information. I'll await your response.

**Comment by MattiaLitrico** [comment]
The hyperparameters for the PACS experiments are the same of VisDA-C experiments.
